general/community/meetings/index.md

```diff
@@ -13,15 +13,15 @@ Developer meetings are open to anyone interested in Moodle development.
 
 :::important
 
-Our next developer meeting will be on Tuesday 18 October 2022 at 08:00 UTC.
-Details: [Developer meeting October 2022](./202210.md)
+Our next developer meeting will be in December (date tbc).
```

Comment:
If you look at the current version of this page, you'll see Andrew also added yesterday a sentence to explain people can contact you: https://moodledev.io/general/community/meetings

Is that something we want to keep or that should be removed?

As you both were working on the file at the same time, maybe it's easier for you to start a new patch with the latest version and add only the missing changes (like the month of the next developer meeting).